Lib/idlelib/configdialog.py

```diff
@@ -1853,6 +1853,219 @@ def save_all_changed_extensions(self):
             self.ext_userCfg.Save()
 
 
+class FontPage:
+
+    def __init__(self, parent, tab_pages, highlight_sample):
+        self.parent = parent
+        self.tab_pages = tab_pages
+        self.highlight_sample = highlight_sample
+        self.create_page_font_tab()
+        self.load_font_cfg()
+        self.load_tab_cfg()
```

Comment:
Loading immediately instead of waiting until all pages are created ~~*should* be ok.~~ requires that self.highlight exist, because set_samples is called.  Making ConfigDialog pass `tab_pages.pages['highlight'].highlight_sample` is messy (such a detail should normally be hidden) but makes the creation order requirement obvious.  As long as there are no circular dependencies, immediate loading is okay, but I can see why something decided to punt on the issue.  (As far as I know, this is the only cross-page dependency.)